module/Olcs/view/dashboard-tables.phtml

```diff
@@ -1,10 +1,17 @@
 <?php
+
 /* @var \Laminas\View\Helper\Navigation\Menu $menu */
 $menu = $this->navigation($this->navigation('navigation')->getContainer()
     ->findBy('id', 'dashboard-licences-applications'))->menu();
 
-echo $menu->setMinDepth(0)->setMaxDepth(0)
-    ->setPartial('partials/tabs-nav');
+echo $menu->setMinDepth(0)
+          ->setMaxDepth(0)
+          ->setPartial('partials/tabs-nav')
+          ->renderPartialWithParams(
+             [
+               'unreadMessageCount' => $this->__varsCache[1]['unreadMessageCount']
```

Comment:
This oddly isn't accessible using __vars so it's even worse than the others.
Need to find another way